src/main/java/frc/robot/hardware/mechanisms/FlywheelSimulation.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.hardware.mechanisms;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.simulation.FlywheelSim;
+import frc.utils.time.TimeUtils;
+
+public class FlywheelSimulation extends MechanismSimulation {
+
+	private final FlywheelSim flywheelSimulation;
+
+	private Rotation2d position;
+	private Rotation2d lastVelocity;
+	private Rotation2d currentVelocity;
```

Comment:
this seems more like a local variable in get position

Reply:
Agreed

Reply:
what do you mean?